qc/workflow_exome.cwl

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "Exome QC workflow"
+requirements:
+    - class: SubworkflowFeatureRequirement
+inputs:
+    bam:
+        type: File
+        secondaryFiles: [.bai]
+    reference:
+        type: File
+        secondaryFiles: [.fai]
+    bait_intervals:
+        type: File
+    target_intervals:
+        type: File
+    roi_intervals:
```

Comment:
I can't recall if we need a separate set of ROI intervals or if we should simply use the target_intervals.  I'm inclined to start with only target_intervals unless there is a need for a third set of intervals (outside of target/bait).